scripts/config.ts

```diff
@@ -10,8 +10,9 @@ const __DEV__ = env === 'development';
 const __PERF__ = !!process.env.PERF;
 const __PROD__ = env === 'production';
 let __BASENAME__ = process.env.DEPLOYBASEPATH
-  ? // This needs a trailing slash or images won't work
-    `/${process.env.DEPLOYBASEPATH}/react-northstar/`
+  ? // This needs a trailing slash or images won't work.
+    // The path is different for standard PR deploy and screener deploy.
+    `/${process.env.DEPLOYBASEPATH}/${process.env.SCREENER_BUILD ? 'react-northstar-screener' : 'react-northstar'}/`
```

Comment:
It seems that we are missing env variable 🤔 

- I checked Screener logs and it tries to open https://fluentuipr.z22.web.core.windows.net/pull/17429/merge/react-northstar-screener/maximize/flex-example-media-card-shorthand/false (path is "react-northstar-screener" ✅ )
- I opened this page (https://fluentuipr.z22.web.core.windows.net/pull/17429/merge/react-northstar-screener/maximize/flex-example-media-card-shorthand/false) and it tries to open assets from "react-northstar" 🔴 
![image](https://user-images.githubusercontent.com/14183168/111317419-730cc380-8664-11eb-98f4-616772c462d7.png)

Reply:
I think that got lost when I reverted then re-added the separate deployment change. (Also to get that in sooner, I made a separate PR #17448 with just that part since it's already an issue.)